src/app/organisms/room/RoomViewHeader.scss

```diff
@@ -24,4 +25,19 @@
     box-shadow: var(--bs-surface-outline);
     outline: none;
   }
-}
\ No newline at end of file
+}
+
+.room-people-drawer-toggle {
+  @include threshold.smaller(tablet) {
+    display: none;
+  }
+}
+
+.room-back-btn {
+  margin-left: -10px;
+  margin-right: 5px;
```

Comment:
please don't use negative margins. also we uses spacing variables(already defined in index.scss) instead of direct values. also use dir.side mixin for side margin/padding